docs/apv.md

```diff
@@ -557,9 +557,9 @@ Onderstaande regels tellen alleen op Narcopolis
 3. Bij overtreding van lid 1 of lid 2 zal een straf worden gegeven van de 1e categorie.
 4. Er mag op binnenkomende vluchten en schepen geschoten worden, die niet afkomstig zijn van de koninklijke marechaussee.
 
-### Artikel N-2 - Landen van helikopers/vliegtuigen
+### Artikel N-2 - Inladen van helikopers/vliegtuigen
 
-1. Het is verplicht om te landen op een helipad, een landingsbaan of in een straal van 50 meter van een helipad/landingsbaan.
+1. Wanneer jij drugs wilt inladen in een vliegtuig of helikopter moet het vliegtuig of de helikopter binnen een straal van 50 meter van een helipad of landingsbaan staan.
```

Comment:
Dus als ik instap en ik heb drugs bij me heb mag het wel?

Reply:
Niek ging ook kijken om te implementeren dat je gewoon niet meer kan inladen als je te ver van deze zones af bent, het gaat vooral om de reden waarom het is toegevoegd. Dat is dat mensen hun heli op een pluk neerzetten en vervolgens dan volladen. Dit is nu niet toegestaan, maar kan je wel met andere redenen op meerdere plekken landen zonder dat je gebonden bent aan helipads.